atom/browser/api/atom_api_protocol.h

```diff
@@ -93,6 +93,11 @@ class Protocol : public mate::TrackableObject<Protocol> {
     net::URLRequestJob* MaybeCreateJob(
         net::URLRequest* request,
         net::NetworkDelegate* network_delegate) const override {
+      // hander not triggered if enabled.
```

Comment:
?

Reply:
Ah yes, that can go, it's just a left over from porting electron/electron#11008